### DISCUSSION TITLE
想问下怎么调都访问不了web怎么办

### DISCUSSION THREAD

**Discussion by Xenes-cyber** [qd-today/qd#385] [Q&A] [answered]
试着用了Docker Compose和Docker 方式了，端口也改过8294试了，防火墙也放通了，docker也显示运行了，但就是访问不了

**Comment by a76yyyy** [comment]
是不是没做好端口映射?

**Reply by Xenes-cyber** [reply, marked as answer]
谢谢大佬抽空回答，问题已经解决了，之前弄青龙的时候，不知道弄了什么，把Docker的网络弄到了“qinglong_default”里面，后续装了个第三方的Docker管理，在里面调回了bridge就好了
